wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ConversationListViewController+NavigationBar/FilterButtonStyleHelper.swift

```diff
@@ -0,0 +1,55 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import UIKit
+
+struct FilterButtonStyleHelper {
```

Comment:
Can this become a `FilterButton.makeActionImage` to simplify the name and keeping the clarity of naming?

Reply:
Done: https://github.com/wireapp/wire-ios/commit/8671b54fb4e8997bdfe77093214b5bde81ea17c0

Reply:
Here I meant also to rename `FilterButtonStyleHelper` to `FilterButton`, but it's up to you